src/core_plugins/metrics/server/lib/vis_data/helpers/bucket_transform.js

```diff
@@ -1,5 +1,7 @@
 import parseSettings from './parse_settings';
 import getBucketsPath from './get_buckets_path';
+import { parseInterval } from 'ui/utils/parse_interval';
```

Comment:
This import needs to be `import { parseInterval } from '../../../../../../ui/public/utils/parse_interval';` Sorry... I forgot that the `ui/utis/...` path only works on the front end code. :(